dashboard/app/views/pd/workshop_mailer/_supply_list.html.haml

```diff
@@ -1,5 +1,5 @@
 %h3
-  What to Bring
+  What to bring
```

Comment:
Spec has this as a capital B

Reply:
Also there is a section on line 9 on compatible OS and Browsers that needs to be added